docs/framework/symfony.md

```diff
@@ -85,9 +85,18 @@ services:
 
 
     #
-    # Registering a Request Handler
+    # Registering a single Request Handler
     #
     App\SomeClient\RequestHandler\ListSomething:
         arguments:
             - '@App\SomeClient\Transport'  
+
+    #
+    # Or register all Request Handlers at once
+    #
+    App\SomeClient\RequestHandler\:
+        resource: '../../RequestHandler/*'
+        public: true
```

Comment:
Does it need to be public for autowiring?

Reply:
I make it public because i create my clients as a package, so this way only the Request Handlers are exposed to the application.

Reply:
`public` means you can directly `$container->get()` on them, which is a bad (service location) practice.
If you mark them as `public: false`, you can still load them in the service configuration for good dependency injection

Reply:
aight, removed it from the docs